### DISCUSSION TITLE
Data Macro Export Location

### DISCUSSION THREAD

**Discussion by alanv73** [joyfullservice/msaccess-vcs-addin#321] [Q&A]
I'm wondering where a table's Data Macros are exported. I've just done a fresh export of a back end database (not including data) and I'm not finding anything in the tdmacros folder despite the existence of both an After Update and an After Delete macro on one of the tables.
VCS 3.4.17

**Comment by joyfullservice** [comment]
@alanv73 - They should be exporting into the `tdmacros` folder.

![image](https://user-images.githubusercontent.com/7254992/165623673-f7306d4d-c9f5-4312-92d0-fd71257e99b4.png)

If they aren't, go ahead and open an issue, and we can take a closer look. (In the issue, it would also be helpful if you could include a small sample database that demonstrates the problem.)

Thanks!
